src/EmployerFinance/SFA.DAS.EmployerFinance/InnerApi/Responses/GetProviderResponse.cs

```diff
@@ -1,10 +1,10 @@
-﻿namespace SFA.DAS.SharedOuterApi.InnerApi.Responses.ProviderCourses
+﻿namespace SFA.DAS.EmployerFinance.InnerApi.Responses
 {
     public class GetProviderResponse
     {
         public int Ukprn { get; set; }
-        public string LegalName { get; set; }
-        public string Website { get; set; }
+        public string Name { get; set; }
```

Comment:
Is this change required?

Reply:
it does appear to need that change.  For some reason, it was awaiting the fields 'LegalName' and 'Website' from roatp inner, but the fields are actually 'name' and 'ContactUrl', hence the change.    And Corey said this endpoint is  called from the EmployerFinance messagehandlers during the MA payment pull.   I presume such calls would get a null value for the unmapped fields at the moment.